ceph-setup-arm64/config/definitions/ceph-setup.yml

```diff
@@ -23,7 +23,7 @@
 
     scm:
       - git:
-          url: git@github.com:ceph/ceph-releases.git
+          url: git@github.com:ceph/ceph.git
```

Comment:
would you mind explaining this change? we use ceph-releases to prevent publishing tags to ceph.git, this would revert that